packages/server/database/migrations/20200113155150-topTier.ts

```diff
@@ -0,0 +1,49 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function(r: R) {
```

Comment:
now that the user's top tier is on the hottest query, it makes sense to persist it in the DB instead of having some 3 table join